src/index.js

```diff
@@ -133,7 +138,8 @@ class Libp2p extends EventEmitter {
       // Add the identify service since we can multiplex
       this.identifyService = new IdentifyService({
         registrar: this.registrar,
-        peerInfo: this.peerInfo,
+        peerId: this.peerId,
+        addresses: this.addresses,
```

Comment:
We'll eventually want to use announce addresses, so this should probably get passed the Address Manager once it's ready, so that we can get the most up to date announce addresses when responding to identify requests.

Reply:
yep, already being done on #612 :)